modules/fedimint-ln-common/src/lib.rs

```diff
@@ -700,7 +694,22 @@ impl TryFrom<Bolt11Invoice> for PrunedInvoice {
             route_hints: invoice.route_hints().into_iter().map(Into::into).collect(),
             min_final_cltv_delta: invoice.min_final_cltv_expiry_delta(),
             expiry_timestamp,
-        })
+        }
+    }
+}
+
+impl TryFrom<Bolt11Invoice> for PrunedInvoice {
```

Comment:
This will conflict with https://github.com/fedimint/fedimint/pull/5391/files#diff-be0f5678fdb2127a114884ac2582db215bfa8e5306dc02d94f2f54426ce9375a but it should be an easy fix